health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewChecklist.js

```
@@ -0,0 +1,316 @@
+                            label={t("UPDATE_CHECKLIST")}
+                            onClick={() => {
+                                history.push(`/${window.contextPath}/employee/campaign/checklist/update?campaignName=${campaignName}&role=${role}&checklistType=${checklistType}`)
+                            }}
+                        />,
+                    ]}
+                    sortFooterChildren={true}
+                >
+                    {/* <PreviewComponent
+              questionsArray={previewData}></PreviewComponent> */}
+
+                    <MobileChecklist questions={previewData} checklistRole={t(`${roleLocal}`)} typeOfChecklist={t(`${checklistTypeLocal}`)}></MobileChecklist>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)